packages/style-engine/src/styles/utils.ts

```diff
@@ -139,9 +141,9 @@ export function generateBoxRules(
  * @return A CSS var value.
  */
 
-export function getCSSValueFromRawStyle(
-	styleValue: string | any
-): string | unknown {
+export function getCSSValueFromRawStyle< CSSStyleValue = string >(
```

Comment:
`CSSStyleValue` is a [defined type](https://developer.mozilla.org/en-US/docs/Web/API/CSSStyleValue) which could be a little bit confusing for others 😅.

Reply:
🤦🏻 **TIL**

I need to call it something. `T` in the docs is a bit confusing. 

`RawStyleValue` or something

Reply:
Or Just `StyleValue` or even `Type`?

Reply:
Changed to `StyleValue` so it looks good in the generated docs.